new_platforms/nuget/build/native/openenclave.targets

```diff
@@ -0,0 +1,126 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <!-- Relative paths to the Open Enclave SDK Libraries within the NuGet Package -->
+    <PropertyGroup>
+        <SgxHwX86Dbg>$(MSBuildThisFileDirectory)..\..\lib\native\v141\sgx\hw\x86\Debug</SgxHwX86Dbg>
```

Comment:
I see it says 141 not 140.
Will this work for VS 2015 apps?

Reply:
We should build for VS15 as well and include that output in the package, yes.